### ISSUE TITLE
train nerf with custom data

### ISSUE THREAD

**Issue by w2kun** [facebookresearch/pytorch3d#820]
Hi,

I noticed that the datasets supported in this nerf implementation have been processed in advance. I want to use my own data to train nerf. Can you provide some help on how to prepare custom training data?

**Comment by patricklabatut** [comment]
Better support for NeRF-style datasets is on our radar. @bottler could likely elaborate on how to prepare your own training data for now.

**Comment by bottler** [comment]
Pick a name for your scene, e.g. `myscene`.

If you have `N` images of shape `[W,H]`, they should be concatenated and saved into a single very tall png of shape `[W,H*N]` with name `data/myscene.png`.

The cameras and data splits should be put into a `dict` as follows and `torch.save`d into a file `data/myscene.pth`.
* "cameras". This should contain camera intrinsics in PyTorch3D format, as a dictionary of the following things
    * "R": camera rotation as a FloatTensor of shape [N,3,3]
    * "T": camera position as a FloatTensor of shape [N,3]
    * "focal_length": a FloatTensor of shape [N,2]
    * "principal_point": a FloatTensor of shape [N,2]
* "splits". This should be a list with three LongTensors (or lists of ints) containing numbers between `0` and `N - 1` which represent the indices of 
    * members of the training set
    * members of the validation set
    * members of the testing set set

**Comment by w2kun** [comment]
Thanks for the reply.
